app/controllers/follows_controller.rb

```diff
@@ -109,10 +109,12 @@ def follow(followable, need_notification: false)
     "already followed"
   end
 
-  def unfollow(followable, need_notification: false)
+  def unfollow(followable, followable_type, need_notification: false)
     user_follow = current_user.stop_following(followable)
     Notification.send_new_follower_notification_without_delay(user_follow, is_read: true) if need_notification
 
+    Follows::DeleteCached.delete(current_user, followable_type, followable.id)
```

Comment:
just a tiny request and then we're on! `Follows::DeleteCached.call` instead of `Follows::DeleteCached.delete` to follow the pattern of most of other services we have

Reply:
that makes sense - thanks!

Reply:
@aitchiss Not sure if you're aware of it but we have a custom generator for services. `rails generate service Follows::DeleteCached`  would have created the following template (and a spec for it):

```ruby
class Follows::DeleteCached
  def self.call
    new.call
  end

  def call
  end
end
```

Reply:
I didn't know! Thanks!